### PR TITLE
MOSIP-13551

### PR DESCRIPTION
**Pull request by dhanendra06**
https://mosip.atlassian.net/browse/MOSIP-13965 will be taken later..

### CODE REVIEW

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/ReprocessorApplication.java [hunk 1]
```diff
@@ -28,8 +27,8 @@ public static void main(String[] args) {
 		workflowEventUpdateVerticle.deployVerticle();
         WorkflowActionApi workflowActionApi = ctx.getBean(WorkflowActionApi.class);
         workflowActionApi.deployVerticle();
-		WorkflowSearchApi workflowSearchApi = ctx.getBean(WorkflowSearchApi.class);
-		workflowSearchApi.deployVerticle();
+		// WorkflowSearchApi workflowSearchApi = ctx.getBean(WorkflowSearchApi.class);
```
Comment (vishwa-vyom) [inline registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/ReprocessorApplication.java]: Please remove this commented lines

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowActionApi.java [hunk 2]
```diff
@@ -108,10 +112,18 @@ public void deployVerticle() {
 
 	}
 
+	/**
```
Comment (vishwa-vyom) [inline registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowActionApi.java]: Please remove this empty comments

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowActionApi.java [hunk 3]
```diff
@@ -108,10 +112,18 @@ public void deployVerticle() {
 
 	}
 
+	/**
+	 * 
+	 */
 	@Override
 	public void start() {
+
```
Comment (vishwa-vyom) [inline registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowActionApi.java]: Remove this unwanted new line

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java [hunk 4]
```diff
@@ -96,35 +74,16 @@
...
-		return Integer.parseInt(eventBusPort);
-	}
 
 	/**
 	 * contains all the routes in this stage
 	 *
 	 * @param router
 	 */
-	private void routes(MosipRouter router) {
+	public void routes(MosipRouter router) {
```
Comment (vishwa-vyom) [inline registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java]: make this method private again

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java [hunk 5]
```diff
@@ -96,35 +74,16 @@
...
 		router.post(contextPath + "/workflow/search");
 		router.handler(this::processURL, this::failure);
+
```
Comment (vishwa-vyom) [inline registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java]: Remove unwanted new line

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java [hunk 6]
```diff
@@ -196,12 +155,9 @@ private void logError(
 	}
 
 	private void failure(RoutingContext routingContext) {
+		regProcLogger.info("failure method is getting called>>>>>>>>>");
```
Comment (vishwa-vyom) [inline registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java]: Remove this log statement

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java [hunk 7]
```diff
@@ -219,6 +175,7 @@ private void buildResponse(RoutingContext routingContext, PageResponseDTO<Workfl
 	}
 
 
+
```
Comment (vishwa-vyom) [inline registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java]: Remove this unwanted new line

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]
Raise a new PR with the comments resovled and with JIRA id of the actual story in the PR title

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/ReprocessorApplication.java (hunk 1 above)
Comment (dhanendra06) [reply registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/ReprocessorApplication.java]: taken care

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowActionApi.java (hunk 2 above)
Comment (dhanendra06) [reply registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowActionApi.java]: taken care

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowActionApi.java (hunk 3 above)
Comment (dhanendra06) [reply registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowActionApi.java]: taken care

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java (hunk 4 above)
Comment (dhanendra06) [reply registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java]: taken care

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java (hunk 5 above)
Comment (dhanendra06) [reply registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java]: taken care

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java (hunk 6 above)
Comment (dhanendra06) [reply registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java]: taken care

File: registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java (hunk 7 above)
Comment (dhanendra06) [reply registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowSearchApi.java]: taken care

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**